configs/kapt/v2/kapt-reseach-v2.md

```diff
@@ -0,0 +1,42 @@
+
+## KAPT Reseacrh with Java/Koltin mix
+
+
+* Date - 2018 - 5 - 21
+* Tools Version
+
+`
+ "buildSystemConfig": {
```

Comment:
Maybe it is better to write it down in plain text, instead of config-like, because `buildSystemVersion `it might be confusing.

Reply:
fixed